src/AnomalyDetectionResult.php

```diff
@@ -126,6 +131,14 @@ public function toArray(int $precision = 2): array
         );
     }
 
+    /**
+     * The number of observations
+     */
+    public function getCount(): int
+    {
+        return $this->count;
+    }
```

Comment:
We would be able to see how many observations there are to make a better decision.